src/api/public/apidocs-new/paths/staging_project_name_staging_projects_staging_project_name_staged_requests.yaml

```diff
@@ -0,0 +1,199 @@
+get:
+  summary: List all the staged requests of a staging project.
+  description: List all the requests that are already associated to a staging project (staged requests).
+  # Authentication is not required
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/staging_project_name.yaml'
+  responses:
+    '200':
+      description: List of staged requests.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/staging/staged_requests.yaml'
+          examples:
+            Multiple Requests:
+              value:
+                request:
+                  - id: 1
+                    creator: Iggy
```

Comment:
This response on my side looks like this:
```
<staged_requests>
  <request id="12" type="submit" creator="Admin" state="review" package="hello_world" superseded_by="" updated="2023-04-28T09:39:14Z"/>
</staged_requests>
```

Reply:
True! I didn't complete the example... done!